custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -1247,6 +1247,24 @@
           "type": "property_name",
           "property_name": "child_name"
         }
+      },
+      {
+        "column_id": "location_reassignment_date",
```

Comment:
I assume all these fields will have data going forward so no need to backfill yeah?

Reply:
exactly